yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1441,6 +1441,7 @@ To manually modify the schema, please refer: <a class="highlight-link" href="htt
 
 	GIN_INDEXES         = `There are some BITMAP indexes present in the schema that will get converted to GIN indexes, but GIN indexes are partially supported in YugabyteDB as mentioned in <a class="highlight-link" href="https://github.com/yugabyte/yugabyte-db/issues/7850">https://github.com/yugabyte/yugabyte-db/issues/7850</a> so take a look and modify them if not supported.`
 	UNLOGGED_TABLE_NOTE = `There are some Unlogged tables in the schema. They will be created as regular LOGGED tables in YugabyteDB as unlogged tables are not supported.`
+	REPORTING_LIMITATIONS_NOTE = `<a class="highlight-link" target="_blank"  href="https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/#assessment-and-schema-analysis-limitations">Limitations in the assessment</a>`
```

Comment:
It's a bit weird to have html in the notes field of the struct, because it will go into the json as well. 
However, I see that we have that for other items already, so it's okay for now, we can revisit this later.

Reply:
Also, let's have `Limitations in assessment` ? (remove `the`)

Reply:
Yes, I also didn't focus on fixing it because that might require extra handling for Notes and we already. had it for other cases. so will take that up later.